EIPS/eip-5XXX.md

```diff
@@ -0,0 +1,341 @@
+---
+eip: 5XXX
```

Comment:
```suggestion
eip: 5089
```
Don't forget to update filename as well.

Reply:
I was doing it!

Reply:
:D

Reply:
Can you update the PR title to include EIP-5089?

Reply:
Can you update the PR title to include EIP-5089?